package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "nativeTemplate",
+  "name": "native-template",
   "version": "0.0.1",
```

Comment:
According to semver, this should default to `0.1.0` as we're adding features atm, not fixing bugs.

Reply:
done!